addon/components/o-s-s/power-select.hbs

```diff
@@ -2,6 +2,7 @@
      {{did-insert (fn this.ensureBlockPresence (has-block 'selected-item') (has-block 'option-item'))}}>
   <div class="upf-power-select__array-container margin-bottom-xxx-sm">
     <div class="array-input-container {{if @selectedItems 'fx-row fx-xalign-center' 'fx-col fx-malign-center'}} {{if this.hasScrollbar 'has-overflow'}}"
+         data-control-name="power-select_array-input-container"
```

Comment:
not sure it's the role of the power-select itself to set this as it depends on the context it's used in (it will be passed in the top `...attributes` by the parent application.